pkg/checkmarxone/checkmarxone.go

```diff
@@ -838,6 +838,26 @@ func (sys *SystemInstance) CreateProjectInApplication(projectName, applicationID
 	}
 
 	err = json.Unmarshal(data, &project)
+	if err != nil {
+		return project, errors.Wrapf(err, "failed to unmarshal project data")
+	}
+
+	// since there is a delay to assign a project to an application, adding a check to ensure project is ready after creation
+	// (if project is not ready, 403 will be returned)
+	projectID := project.ProjectID
+	project, err = sys.GetProjectByID(projectID)
+	if err != nil {
+		const max_retry = 180 // 3 minutes
+		const delay = 1
```

Comment:
delay is too low, this can tamper the system. Can we increase the delay to 60.
And max_retry to 15 or 30.

Reply:
fixed, thanks